runtime/onert/backend/cpu/ops/SpaceToBatchNDLayer.cc

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "SpaceToBatchNDLayer.h"
+
+#include "OperationUtils.h"
+
+#include <cker/operation/SpaceToBatchND.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace cpu
+{
+namespace ops
+{
+SpaceToBatchNDLayer::SpaceToBatchNDLayer()
+    : _input(nullptr), _block_shape(nullptr), _padding(nullptr), _output(nullptr)
+{
+  // DO NOTHING
+}
+
+// TO DO : move into shape inferer
+void SpaceToBatchNDLayer::checkDimension()
+{
+  const int kSpatialDimensionNum = 2;
```

Comment:
The value for `kSpatialDimensionNum` is set to 2, which seems to deviate from TFlite implementation where the corresponding value is `input_size->size -2`. Or is it that  both expressions are similar? Please ignore comment if trivial.

Reply:
This is a limitation on tflite kernel. You can find in tensorflow operation spec.  It only support like this :  
- input's rank must be 4 (ex, [1, 4, 4, 8])
- spatial dimension must be size of 2 (ex, [2,2])